src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.hellbender;
+
+import org.broadinstitute.hellbender.utils.NativeUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+public class JBWAIntegrationTest extends BaseTest {
+
+    @Test
+    public void testJBWAIsLoadable(){
+        if ( ! NativeUtils.runningOnLinux() && ! NativeUtils.runningOnMac() ) {
+            throw new SkipException("jbwa not available on this platform");
```

Comment:
what does this have to do with jbwa? this seems to be a generic check not related to jbwa